stacks/spica/src/function/pages/log-view/log-view.component.html

```diff
@@ -43,6 +43,9 @@ <h4 class="mat-h4">
       </mat-option>
     </mat-select>
   </mat-form-field>
+  <mat-slide-toggle [(ngModel)]="showErrors" (ngModelChange)="filter$.next(filter$.value)">
+    Hide Errors
```

Comment:
When this toggle switched true, it will display the errors but this text still stays as **Hide Error** which means the opposite of current behaviour. It should be **Show Errors** instead of **Hide Errors** or it should change when toggle switched.